force-app/main/default/lwc/formattedDocsViewer/__tests__/formattedDocsViewer.test.js

```diff
@@ -0,0 +1,55 @@
+import { createElement } from 'lwc';
+import FormattedDocsViewer from 'c/formattedDocsViewer';
+import { loadScript, loadStyle } from 'lightning/platformResourceLoader';
+
+const mockFetch = (data) => {
+    return jest
+        .fn()
+        .mockImplementation(() =>
```

Comment:
This can be simplified with a sugar syntax `jest.fn().mockResolvedValue({ ok: true, text: () => Promise.resolve(data) })`

Reply:
Yeah, but I really like the readability of the multiline version.